wallet-connect/.yarnrc.yml

```diff
@@ -0,0 +1,5 @@
+nodeLinker: node-modules
+
+pnpMode: loose
+
+yarnPath: .yarn/releases/yarn-3.2.0.cjs
```

Comment:
this is also already in the root. Why adding another one? We are using monorepos

Reply:
Removed